src/utils/helpers.ts

```diff
@@ -43,17 +42,8 @@ export function calculateDelay(delay: DelayType, retryCount: number): number {
     return typeof delay === 'function' ? delay(retryCount) : delay;
 }
 
-export function generateQueueName(handler: Type<AbstractPubsubAnyEventHandler>): string {
-    if (!process.env.npm_package_name) {
-        throw new Error(
-            'The application should be started using npm scripts. Otherwise npm_package_name environment variable which is required for queue name generation is missing',
-        );
-    }
-
-    const pckName: string = process.env.npm_package_name.split('/').pop() as string;
-    const platform = pckName.replace(/[_-]/gi, '.');
-
-    return [platform, toSnakeCase(handler.name)].join(':');
+export function generateQueuePrefixFromPackageName(): string | undefined {
+    return process.env.npm_package_name?.split('/')[0].replace(/[_-]/gi, '.');
```

Comment:
check somethink like
`return process.env.npm_package_name?.split('/')?.[0]?.replace(/[_-]/gi, '.');`

Reply:
`split` always returns an array with minimum one element

Reply:
There is a question mark after `process.env.npm_package_name`